src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -1,3 +1,20 @@
+<script setup lang="ts">
+import Breadcrumb from './Breadcrumb/Breadcrumb.vue';
+import type { BreadcrumbType } from './Breadcrumb/types';
+
+defineProps({
+  items: {
+    type: Array as () => Array<BreadcrumbType>,
```

Comment:
I think you are supposed to use `PropType`

Reply:
Done